Formula/git-tidy.rb

```diff
@@ -0,0 +1,18 @@
+class GitTidy < Formula
+  desc "Delete git branches that have a remote tracking branch that is : gone"
+  homepage "https://github.com/drewwyatt/git-tidy"
+  url "https://github.com/drewwyatt/git-tidy/archive/1.0.0.tar.gz"
+  sha256 "5bdae85d8ac4fe59305781846229017a8649d894f82ed9cba243ea1a534bd511"
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-X main.version=#{version}"
+    bin.install "git-tidy"
+  end
+
+  test do
+    output = `git tidy -v`
+    expected = "#{version}\n"
+    assert_equal expected, output
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.